docs/product/components/cards.html

```diff
@@ -9,13 +9,18 @@
     <p class="stacks-copy">The base card styling applies a border and padding to the card.</p>
     <p class="stacks-copy">Cards can be any size and it’s ok to increase the body text size for larger cards.</p>
     <div class="stacks-preview">
+        <div class="stacks-clipboard-content ps-relative" data-controller="clipboard">
```

Comment:
We could/should probably extend/replace the `highlight` shortcode to include the copy/paste button as well. That way we'd get it "for free" and wouldn't have to add boilerplate all over the docs.

Reply:
I'd like to add the copy button to the `highlight` shortcode, but also have it available standalone. This will allow us to add a copy button to any element, like so:

![image](https://user-images.githubusercontent.com/647177/177812404-b26c644a-ec53-406d-a554-fdb024505b8b.png)

It's a contrived example, but I think it illustrates the point.